### PR TITLE
I3S-Debug - Add LOD + OBB Validation

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

File: examples/experimental/i3s-17-and-debug/tile-debug.js [hunk 1]
```diff
@@ -61,3 +65,117 @@ function getChildrenInfo(children) {
...
+
+  for (let i = 0; i < positions.length; i += 3) {
+    const positionsVector = new Vector3(
+      (positions[i] *= halfSize[0]),
+      (positions[i + 1] *= halfSize[1]),
+      (positions[i + 2] *= halfSize[2])
+    );
+    const rotatedPositions = positionsVector
+      .transformByQuaternion(tile.header.obb.quaternion)
+      .add(obbCenterCartesian);
```
Comment (belom88) [inline examples/experimental/i3s-17-and-debug/tile-debug.js:160]: You stay in cartesian ECEF so this calculations look correct for me.

File: examples/experimental/i3s-17-and-debug/tile-debug.js [hunk 2]
```diff
@@ -61,3 +65,117 @@ function getChildrenInfo(children) {
...
+
+  return vertices;
+}
+
+function isAllObbVerticesInsideParentObb(parentObb, tilePositions) {
+  let isTileObbInsideParent = true;
+
+  for (let index = 0; index < tilePositions.length / 3; index += 3) {
+    const point = [tilePositions[index], tilePositions[index + 1], tilePositions[index + 2]];
+    const distance = parentObb.distanceTo(point);
```
Comment (belom88) [inline examples/experimental/i3s-17-and-debug/tile-debug.js:172]: Does this `distanceTo` consume absolute cartesian (ECEF) coordinates or cartographic instead? You have them here in the `point` vector.

File: examples/experimental/i3s-17-and-debug/app-debug.js [hunk 3]
```diff
@@ -249,6 +251,15 @@ export default class App extends PureComponent {
     this.setState({debugOptions});
   }
 
+  handleValidateTile(tile) {
```
Comment (belom88) [inline examples/experimental/i3s-17-and-debug/app-debug.js:254]: Nit: `handleTileValidation`

**Review by belom88 — APPROVED** [review]
We possibly can merge it along with https://github.com/visgl/loaders.gl/pull/1299 as I need you "TODO" functions from this PR.

File: examples/experimental/i3s-17-and-debug/app-debug.js (hunk 3 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/app-debug.js:254]: I think just `validateTile` will be better. You thoughts?

File: examples/experimental/i3s-17-and-debug/tile-debug.js (hunk 2 above)
Comment (dryabinin-actionengine) [reply examples/experimental/i3s-17-and-debug/tile-debug.js]: Added point transformation from `ECEF -> Cartographic ` before measuring distance.

**Pull request merged**